.github/workflows/cd-pipeline.yaml

```
@@ -141,3 +142,43 @@ jobs:
           git config user.name "Bitnami Containers"
           git config user.email "bitnami-bot@vmware.com"
           git add bitnami/index.yaml && git commit -m "${chart}-${chart_version}: Update index.yaml" -s && git push
+
+  # If the CD Pipeline does not succeed we should notify the interested agents
+  slack-notif:
+    runs-on: ubuntu-latest
+    needs: vib-publish
+    if: always()
+    name: Notify unsuccessful CD run
+    steps:
+      - name: Notify in Slack channel
+        if: ${{ needs.vib-publish.result != 'success' }}
+        uses: slackapi/slack-github-action@v1.19.0
+        with:
+          channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
```

I would change the secret name to include something related to the CD process:

```suggestion
          channel-id: ${{ secrets.CD_SLACK_CHANNEL_ID }}
```

---

Done! Also modified the ENVs names in the repo settings.